src/models.py

```diff
@@ -496,11 +496,12 @@ def build_image_sent_module(task, d_inp, params):
     return pooler
 
 
-def build_single_sentence_module(task, d_inp, params):
+def build_single_sentence_module(task, d_inp, use_bert, params):
     ''' Build a single classifier '''
-    pool_type = "max"
-    pooler = Pooler(project=True, d_inp=d_inp, d_proj=params['d_proj'], pool_type=pool_type)
-    classifier = Classifier.from_params(params['d_proj'], task.n_classes, params)
+    pool_type = "first" if use_bert else "max"
```

Comment:
Is this behavior documented somewhere? (also seems like this should be configurable)

Reply:
It's in the README now. Yea, it probably should be configurable, but I'll leave it to another PR for now.

Reply:
SG. Maybe add a TODO?

Reply:
:+1:, done